internal/store/json.go

```diff
@@ -239,30 +245,46 @@ func newStoreFromV0(storeStruct storeV0, arch distro.Arch) *Store {
 	return &store
 }
 
+func newBlueprintsV0(blueprints map[string]blueprint.Blueprint) blueprintsV0 {
+	blueprintsStruct := make(blueprintsV0)
+	for name, blueprint := range blueprints {
+		blueprintsStruct[name] = blueprint.DeepCopy()
+	}
+	return blueprintsStruct
+}
+
+func newWorkspaceV0(workspace map[string]blueprint.Blueprint) workspaceV0 {
+	workspaceStruct := make(workspaceV0)
+	for name, blueprint := range workspace {
+		workspaceStruct[name] = blueprint.DeepCopy()
+	}
+	return workspaceStruct
+}
+
+func newComposeV0(compose Compose) composeV0 {
+	return composeV0{
+		Blueprint: compose.Blueprint, // TODO: deep copy
```

Comment:
Why is this TODO? :) Is there any problem with calling `compose.Blueprint.DeepCopy()`?

Reply:
This looks like an oversight, I'll fix!